src/clarity-angular/popover/signpost/signpost-trigger.directive.ts

```diff
@@ -40,10 +43,16 @@ export class SignpostTriggerDirective implements OnDestroy {
      * @function onSignpostTriggerClick
      *
      * @description
-     * click handler for the Signpost trigger button used to hide/show SignpostContent.
+     * Enter key, Space key and click handlers for the Signpost trigger button are used to hide/show SignpostContent.
+     *
+     * The enter HostListener handles the case when the directive is on a non-standard element e.g clr-icon without
+     * a button wrapper and we preventDefault to keep the page from scrolling when the Space key is used.
```

Comment:
I dont think we can do this. If users use it without the button wrapper then its a wrong way of using Signposts. Listening to the space key instead and changing the default behavior might cause unexpected behavior if the user is expecting to scroll the page using a spacebar